webapp/src/translations/languages/en.json

```diff
@@ -220,7 +220,7 @@
     "tokens": {
       "tokensPage": {
         "title": "Tokens",
-        "loading": "Loading…",
+        "loading": "Loading ...",
```

Comment:
"Loading..." should remain as "Loading...", no `space`. However, the `.` used is correct.

Reply:
You are right. I did some research and removed the space before the ellipsis.